### ISSUE TITLE
Typos in accessible commands view

### ISSUE THREAD

**Issue by joaomoreno** [microsoft/vscode#208653]
Testing #207637

There's a double close parenthesis here:

<img width="305" alt="image" src="https://github.com/microsoft/vscode/assets/22350/5775c82e-706c-4618-b2da-1aacea2305eb">

**Comment by joyceerhl** [comment]
Also a missing `.` on the `Go to a symbol` line
